frontend/src/static/js/v_summary.js

```diff
@@ -706,11 +705,12 @@ window.vSummary = {
 
     openTabZoom(user, since, until, repo, index) {
       const { avgCommitSize } = this;
-
+      const clonedUser = Object.assign({}, user); // so that changes in summary won't affect zoom
```

Comment:
Changes to `v_authorship/zoom` looks good.
I'm not too familiar with `v_summary` to know if there are any other issues here though =X

Just one question: this is what fixes #1145 right?

Reply:
Yep, changes in summary won't affect zoom anymore.